.pre-commit-config.yaml

```diff
@@ -88,15 +101,17 @@ repos:
         additional_dependencies:
           - flake8-docstrings
   - repo: https://github.com/PyCQA/isort
-    rev: 5.8.0
+    rev: 5.9.2
     hooks:
       - id: isort
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.812
+    rev: v0.910
     hooks:
       - id: mypy
+        additional_dependencies:
+          - types-setuptools
```

Comment:
Should we also add these dependencies to `requirements-test.txt` or `requirements-dev.txt` so that `mypy` also works from the developer's local venv?

Reply:
I think it'd be cleaner to leverage `mypy --install-types --non-interactive src/` in `setup-env` to automatically install them to the local environment as part of setup.

Reply:
Since that also goes through type checking, we could eschew the `--non-interactive` part (which will also force a user to interact with the `setup-env` script during setup).

Reply:
> I think it'd be cleaner to leverage `mypy --install-types --non-interactive src/` in `setup-env` to automatically install them to the local environment as part of setup.

OK, that makes sense.  That's a change we'd want to make upstream, in skeleton-generic right?

Reply:
> Since that also goes through type checking, we could eschew the `--non-interactive` part (which will also force a user to interact with the `setup-env` script during setup).

I don't understand what you're saying here.

Reply:
Sorry the `--non-interactive` step will also proceed with `mypy`'s type checking. Without that flag you have to do the `--install-types` step and *then* run `mypy` to do type checking. People wanted to do it all at once, but for our purposes it's ok to keep it broken up I believe.

Reply:
> 
> 
> > I think it'd be cleaner to leverage `mypy --install-types --non-interactive src/` in `setup-env` to automatically install them to the local environment as part of setup.
> 
> OK, that makes sense. That's a change we'd want to make upstream, in skeleton-generic right?

Currently `mypy` is only a development requirement for projects descending from this skeleton. We can either add it here or add it to [cisagov/skeleton-generic](https://github.com/cisagov/skeleton-generic) and add `mypy` as a requirement as well.

Reply:
> > I think it'd be cleaner to leverage `mypy --install-types --non-interactive src/` in `setup-env` to automatically install them to the local environment as part of setup.
> 
> OK, that makes sense. That's a change we'd want to make upstream, in skeleton-generic right?

Never mind, I see now that `mypy` isn't even installed in skeleton-generic.

Reply:
See commit 3452b9d.